lib/grpc/GrpcService.ts

```diff
@@ -294,6 +294,21 @@ class GrpcService {
     }
   }
 
+  /**
+   * See [[Service.getNodeReputation]]
+   */
+  public getNodeInfo: grpc.handleUnaryCall<xudrpc.GetNodeInfoRequest, xudrpc.GetNodeInfoResponse> = async (call, callback) => {
+    try {
+      const { banned, reputationScore } = await this.service.getNodeInfo(call.request.toObject());
+      const response = new xudrpc.GetNodeInfoResponse();
+      response.setBanned(banned);
+      response.setReputationscore(reputationScore);
+      callback(null, response);
+    } catch (error) {
```

Comment:
Minor nit but elsewhere in the file we are using `err` instead of `error` if you wouldn't mind changing.